### DISCUSSION TITLE
Example code for Webpack Code-Splitting of translation files

### DISCUSSION THREAD

**Discussion by SimeonC** [i18next/i18next#1611] [Show and tell]
Hey all 👋 

I wrote up this gist for how to code split translations using webpack and wanted to share it. There doesn't seem to be any easy solution code around for this and there's no plugin, probably because it's too simple and setup specific to be a plugin.

I coded this up into one of our projects recently to make the JS bundle much smaller, hope it can help someone else. I thought that maybe this could be a good addition to the docs as it's a very simple custom plugin and I don't think this particular use case is really covered anywhere.

https://gist.github.com/SimeonC/6a738467c691eef7f21ebf96918cd95f

**Comment by jamuhl** [comment]
guess yes - it's to simple - but still -> feel free to add it to the list as an example https://www.i18next.com/overview/plugins-and-utils#backends

**Reply by SimeonC** [reply]
Thanks, I opened a PR for adding it to that section; https://github.com/i18next/i18next-gitbook/pull/160

**Reply by jamuhl** [reply]
was merged...thank you
